.circleci/config.yml

```diff
@@ -16,9 +16,7 @@ jobs:
 
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package-lock.json" }}
-            # fallback to using the latest cache if no exact match is found
```

Comment:
Fallbacking to the latest cache when there is no cache with exact match was the cause of circleci check failure,